pkg/credentials/static_test.go

```diff
@@ -1,6 +1,6 @@
 /*
- * Minio Go Library for Amazon S3 Compatible Cloud Storage
- * (C) 2017 Minio, Inc.
+ * Copyright
+ *  2017 Minio, Inc.
```

Comment:
doesnt look right, should be:
Minio Go Library for Amazon S3 Compatible Cloud Storage
Copyright 2017 Minio, Inc.